sources/3.0.0/sections/10-data_product_format.adoc

```diff
@@ -232,6 +232,8 @@ Example: INT.IHO.S-102.3.0.0
 .4+| The values are in decimal degrees. +
 If a projected CRS is used for the dataset, these values refer to those of the baseCRS underlying the projected CRS (see <<note3>>).
 
+The root bounding box needs to encompass all data. Including no fill values. The outermost cell boundaries of the grid cells and the bounding box / domainExtent.polygon of each feature instance group form the basis for the root bounding box.
```

Comment:
"The root bounding box needs to encompass all data, including no fill values." to clarify that the inclusion of the no fill values is referred to "all data".

Reply:
"The bounding box and the domain extent polygon of each feature instance group form the basis for the root bounding box." because the previous sentence already takes care of the cells.

Reply:
> "The bounding box and the domain extent polygon of each feature instance group form the basis for the root bounding box." because the previous sentence already takes care of the cells.

I wanted to mention the outer cell boundaries because they are also explicitly mentioned in other sections of the PS. Is it ok for you if I leave this part as it is?

Reply:
That's fine. Can we at least use "domain extent polygon" in place of "domainExtent.polygon"? Mainly for being aligned with "the bounding box".